shaders/circle.fragment.glsl

```diff
@@ -10,4 +10,8 @@ varying lowp float v_antialiasblur;
 void main() {
     float t = smoothstep(1.0 - max(u_blur, v_antialiasblur), 1.0, length(v_extrude));
     gl_FragColor = v_color * (1.0 - t) * u_opacity;
+
+#ifdef OVERDRAW_INSPECTOR
+    gl_FragColor = vec4(1.0);
+#endif
```

Comment:
Is there any way to avoid setting `gl_FragColor` and then overwriting `gl_FragColor`?

 - changing the color and opacity passed in the `draw_*` methods
 - putting the other set in a `#ifndef` block